config/swagger.yaml

```diff
@@ -2399,7 +2401,7 @@ definitions:
       name_hash:
         type: string
       recipient_pubkey:
```

Comment:
This property name may be misleading (pubkey while it can be name)?

Reply:
this is what this property is